apps/web/src/pages/index.tsx

```diff
@@ -1,23 +1,71 @@
 import { Layout } from "../helper/layout";
 import type { ReactElement } from "react";
+import Image from "next/image";
+import Link from "next/link";
 
 export default function Landing() {
   return (
-    <>
+    <div className="flex flex-col h-[calc(100vh-6rem)] justify-between content-start gap-9">
       <div>
-        <h1 className="text-9xl font-sora">
-          <span>SK</span>
-          <span className="text-techdev-gradient-radial">TechDev</span>
-        </h1>
-        {/* <Button variant="contained" color="primary">
-          Join us
-        </Button> */}
+        <div>
+          <h1 className="text-9xl font-sora">
+            <span>SK</span>
+            <span className="text-techdev-gradient-radial">TechDev</span>
+          </h1>
+        </div>
+        <div>
+          <h2 className="text-5xl font-sora leading-[3.55rem] w-[60%]">
+            Technology <br /> meets entertainment.
+          </h2>
+          <div className="flex space-x-2 mt-6">
+            <Link href={"/contact"}>
+              <a>
+                <button className="bg-light-primary text-light-on-primary py-2 px-6 rounded-full dark:bg-dark-primary dark:text-dark-on-primary cursor-pointer hover:opacity-80 transition-all duration-300">
+                  <span className=" text-sm font-sora">Join Us</span>
+                </button>
+              </a>
+            </Link>
+            <Link href={"/about"}>
+              <a>
+                <button className="text-light-primary dark:text-dark-primary py-2 px-6 rounded-full cursor-pointer border border-light-outline dark:border-dark-outline hover:opacity-80 transition-all duration-300">
+                  <span className="text-sm font-sora">Learn More</span>
+                </button>
+              </a>
+            </Link>
+          </div>
+        </div>
       </div>
-    </>
+      <div className="flex justify-center mb-6">
+        <Image
+          src="/landing graphic.svg"
+          alt="Landing Graphic"
```

Comment:
This should probably be a bit more descriptive. You're treating accessibility like it's a checkbox to tick off.